Documentation/kubernetes/policy.rst

```diff
@@ -18,6 +18,10 @@ with Kubernetes:
   `ThirdPartyResource` and `CustomResourceDefinition` which supports to specify
   policies at Layers 3-7 for both ingress and egress.
 
+It is recommended to only use one of the above policy types at a time to
+minimize unintended effects arising from the interaction between the
```

Comment:
Do you have something specific in mind? I would probably stick to one as well but nothing specific comes to mind right now.

Reply:
I don't have specific poor interactions in mind, it's more just that there are then two different commands to fetch the policies and this could create confusion. For instance, an L4 k8s NetworkPolicy could end up shadowing an L4+L7 CiliumNetworkPolicy, and there would appear to be a mismatch between the forwarding behaviour and the policy being specified.